### DISCUSSION TITLE
Lỗi chặn quảng cáo trên youtube

### DISCUSSION THREAD

**Discussion by beu2k2** [bigdargon/hostsVN#345] [Q&A] [answered]
Bị lỗi youtube
![CA7A3BCE-AEC2-4981-82A4-B50DF02D78D6](https://github.com/bigdargon/hostsVN/assets/133527985/a6b50f55-1df3-4f39-b3e1-db636d615fd5)

**Comment by bigdargon** [comment]
@beu2k2 bạn cung cấp thêm thông tin đang sử dụng app nào để chặn qc để mình kiểm tra

**Reply by beu2k2** [reply]
Shadow rocket

**Reply by beu2k2** [reply]
Spotify vẫn chặn bình thường mỗi youtube là bị lỗi

**Comment by beu2k2** [comment]
Vẫn chưa khắc phục được ạ ?

**Reply by bigdargon** [reply, marked as answer]
@beu2k2 mấy bữa nay mình bận nhiều việc nên chưa trả lời bạn kịp. Mình kiểm tra dường như bạn chưa tin cậy chứng chỉ của Shadowrocket.

Bạn vào Cài đặt > Cài đặt chung > Thông tin thiết bị > Cài đặt chứng chỉ tin cậy > Bật tin cậy chứng chỉ của Shadowrocket lên

![E41FE2D9-6F7B-4676-9F4A-776832E155B7](https://github.com/bigdargon/hostsVN/assets/10969626/26332831-9000-47ba-8c9d-f523ebf7ec29)

**Comment by beu2k2** [comment]
![Uploading IMG_2702.png…]()

**Comment by beu2k2** [comment]
![Uploading IMG_2703.png…]()
